src/python/pants/backend/python/goals/lockfile_test.py

```diff
@@ -111,14 +120,35 @@ def test_pex_lockfile_generation(
     rule_runner: RuleRunner, no_binary: bool, only_binary: bool
 ) -> None:
     args = ["--python-resolves={'test': 'foo.lock'}"]
+    only_binary_lock_str = '//   "only_binary": [],\n'
```

Comment:
Do `only_binary` / `no_binary` handle name normalization (I suspect not since the metadata types are `set[str]`)? If not there is over-invalidation happening when someone switches from `pantsbuild.pants` to `pantsbuild-pants` even though both are the same and do not affect resolves.

Reply:
They do not. Good point. I'll fix in a follow up :) https://github.com/pantsbuild/pants/issues/16527